.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,116 @@
+---
+name: CI and CD
+
+on:
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+        type: choice
+        options:
+        - info
+        - warning
+        - debug
+      tags:
+        description: 'Test scenario tags'
+        required: false
+        type: boolean
+      environment:
+        description: 'Environment to run tests against'
+        type: environment
+        required: true
+  push:
+    branches: ['main']
+  pull_request:
+    branches: ['main']
```

Comment:
Do you mean to update this to the v1 branch name? Are the checks working with these updates?

Reply:
Good catch! updated here: 7cbb142642cb903e60135a0bfd03e7f205549a86